turbopack/crates/turbopack-core/src/source_map/mod.rs

```diff
@@ -200,6 +200,11 @@ impl SourceMap {
         SourceMap::Decoded(InnerSourceMap::new(map))
     }
 
+    /// Creates a new SourceMap::Decoded Vc out of a [InnerSourceMap] instance.
+    pub fn new_inner(map: InnerSourceMap) -> Self {
+        SourceMap::Decoded(map)
+    }
+
```

Comment:
```suggestion
    /// Creates a new SourceMap::Decoded Vc out of a [InnerSourceMap] instance.
    pub fn new_inner(map: InnerSourceMap) -> Self {
        SourceMap::Decoded(map)
    }

```
We don't need that anymore

Reply:
```suggestion
```